jmclient/jmclient/configure.py

```diff
@@ -986,6 +991,9 @@ def is_native_segwit_mode():
         return False
     return jm_single().config.get('POLICY', 'native') != 'false'
 
+def display_ypub_zpub():
+    return jm_single().config.get('POLICY', 'display_ypub_zpub') != 'false'
```

Comment:
In case setting is renamed as per comment above, function should be renamed too, of course.

Reply:
Done